### PR TITLE
Add clarification that auto-cancellation happens after spec

### PR DESCRIPTION
**Pull request by debrisapron**
Resolves Jira ticket CYCLOUD-754

### CODE REVIEW

**Comment by debrisapron** [comment]
New content preview here: https://cypress-documentation-4dcrch23b-cypress-io.vercel.app/guides/cloud/smart-orchestration#Auto-Cancellation

**Review by rockhold — APPROVED** [review]
LGTM 👍🏼

File: content/guides/cloud/smart-orchestration.md [hunk 1]
```diff
@@ -87,7 +87,9 @@ Continuous Integration (CI) pipelines are typically costly processes that can
 demand significant compute time. **When a test failure occurs in CI, it often
 does not make sense to continue running the remainder of a test suite** since
 the process has to start again upon merging of subsequent fixes and other code
-changes.
+changes. When Auto Cancellation is enabled, once the number of failed tests goes
+over a preset threshold, the entire test run is cancelled. Note that any
```
Comment (ryanpei) [inline content/guides/cloud/smart-orchestration.md:91]:
```suggestion
over a preset threshold, the entire test run is canceled. Note that any
```
We've used `canceled` throughout the docs, and that aligns with American English convention.

**Review by ryanpei — CHANGES_REQUESTED** [review]
minor spelling edit

**Pull request merged**